using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -103,116 +119,150 @@ This API synchronizes the provided customer identifier type key and value with t
+### syncIdentifiers \(overloaded\) 
 
-This API synchronizes the provided customer identifiers to the Adobe Experience Cloud ID Service. If a customer ID type matches an existing ID type, the customer ID is updated with the new ID value and [authentication state](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/identity/identity-api-reference#authenticationstate). New customer IDs are added. These IDs are preserved between app upgrades, are saved and restored during the standard application backup process, and are removed at uninstall. If the current SDK privacy status is `optedout`, calling this method results in no operations being performed.
+This  API  `syncIdentifiers()` is an overloaded version that does not include the parameter for the Authentication State, and assumes a default value as `ACPMobileVisitorAuthenticationStateUnknown`
 
-{% hint style="info" %}
-The `identifiers` dictionary contains IDs with the Identifier type as the key, and the string identifier as the value.
-{% endhint %}
-
-### **Syntax**
+#### Syntax
 
 ```objectivec
-+ (void) syncIdentifiers: (nullable NSDictionary*) identifiers          
-         authentication: (ACPMobileVisitorAuthenticationState) authenticationState;
++ (void) syncIdentifiers: (nullable NSDictionary*) identifiers;
 ```
 
-### **Examples**
+o  *identifiers* contains a dictionary of Identifiers with the Identifier type as the key, and the string identifier as the value. If the identifier type and identifier do not have valid string values \(non null or empty\), the identifier is ignored by the Identity extension.
+
+#### Examples
 
 **Objective-C**
 
 ```objectivec
-NSDictionary *ids = @{@"idType":@"idValue"};
-[ACPIdentity syncIdentifiers:ids authentication:ACPMobileVisitorAuthenticationStateAuthenticated];
+NSDictionary *ids = @{@"idType1":@"idValue1", 
+                      @"idType2":@"idValue2", 
+                      @"idType3":@"idValue3"};
+[ACPIdentity syncIdentifiers:ids];
 ```
 
 **Swift**
 
 ```swift
-let identifiers : [String: String] = ["idType1":"idValue1", "idType2":"idValue2"];ACPIdentity.syncIdentifiers(identifiers, authentication:
-ACPMobileVisitorAuthenticationState.authenticated)
+let identifiers : [String: String] = ["idType1":"idValue1",
+                                      "idType2":"idValue2",
+                                      "idType3":"idValue3"];
+ACPIdentity.syncIdentifiers(identifiers)
 ```
+
 {% endtab %}
 
 {% tab title="React Native" %}
 #### JavaScript
 
-### syncIdentifier
+### syncIdentifier<a id="syncIdentifier-js"></a>
 
-Updates the provided customer ID with the Adobe Experience Cloud ID Service.
+This `syncIdentifier()` API updates or appends the provided customer identifier type key and value with the given authentication state to the Adobe Experience Cloud ID Service. If the given customer ID type already exists in the service, then it is updated with the new ID and authentication state. Otherwise a new customer ID is added.
```

Comment:
- Please reword to "This API updates or..."
- In the 1st sentence, replace "given" with "selected".
- In the 2nd sentence, replace "given customer ID" with "specified customer ID".